### PR TITLE
Update navbar and footer styling

### PR DESCRIPTION
**Pull request by scottpenneman**
I updated the body tag on all pages for consistency and to allow for the "footer-spacer" div to be able to push the footer to the bottom of the page if needed. It was an easy solution that worked with bootstrap.

There were also a couple of navbar tweaks I made. Heather pointed out we were using container-fluid, which added 15px margin to each end, I removed that so it is 100% width now. Also, the form element for logout on the navbar caused a minor spacing issue, so I hid the element with CSS.

### CODE REVIEW

File: src/main/resources/templates/admin.html [hunk 1]
```diff
@@ -40,9 +40,10 @@
     </li>
 </ul>
 
-<footer th:replace="fragments :: footer"></footer>
-
 </div>
 
+<div th:replace="fragments :: footer-spacer"></div>
+<footer th:replace="fragments :: footer"></footer>
+
```
Comment (scottpenneman) [inline src/main/resources/templates/admin.html:47]: You'll see this in most of the files, where I added the footer-spacer fragment and the footer outside of the main content div.

File: src/main/resources/templates/admin.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     </style>
 </head>
 
-<body>
+<body class="d-flex flex-column min-vh-100">
```
Comment (scottpenneman) [inline src/main/resources/templates/admin.html:13]: This is also in most of the files, where you can designate a flexbox container on the body that allows the footer-spacer to work properly.

**Review by MulesKC — APPROVED** [review]
Looks great! Good catch on the logout form causing that spacing, I had noticed it but had no idea how to fix it.

**Review by hstrodtman — APPROVED** [review]
Looks good!

**Comment by KBeas21** [comment]
I know this is completed, so no reason to make a change but was there a reason we created a new fragment for the footer spacer instead of having the spacer inside of the footer fragment? Just trying to learn for my understanding.

**Comment by scottpenneman** [comment]
> I know this is completed, so no reason to make a change but was there a reason we created a new fragment for the footer spacer instead of having the spacer inside of the footer fragment? Just trying to learn for my understanding.
> 
> 

It could be a result of overthinking... I’ll have to try it with those classes in the footer.

**Comment by KBeas21** [comment]
There is no “right” answer. If you want to try to learn that is fine but other than that there is no big reason to change the way you have it now. It’s just as easy to change the footer spacer over the footer spacer css itself.

**Pull request merged**